packages/caravan-wallets/src/trezor.ts

```diff
@@ -130,9 +130,8 @@ const TREZOR_DEV =
 
 try {
   TrezorConnect.init({
-    connectSrc: TREZOR_DEV
-      ? TREZOR_CONNECT_URL
-      : "https://connect.trezor.io/9.1.9/", // pinning to this connect version to avoid backwards incompatible changes
+    connectSrc: TREZOR_DEV ? TREZOR_CONNECT_URL : undefined,
+    coreMode: "auto", // default value still required to enable WebUSB support
```

Comment:
So what is the default behavior of this change? would it go back to using the latest? I'd rather if we could continue pinning to a particular version if possible. I'm also not familiar with the mechanics of the recent api change though.

Reply:
I believe `connectSrc` is mainly intended for operating on different origins rather than pinning versions, at least I've never come across it being used that way, though I'm not aware of the reason why this was originally implemented. I would have assumed the package and `connectSrc` versions should generally remain in lockstep.

Reply:
> mainly intended for operating on different origins rather than pinning versions, at least I've never come across it being used that way,

Heh, well you've come across one instance of it being used that way here! The rationale was explained [here](https://github.com/caravan-bitcoin/caravan/issues/161#issuecomment-2541827797), but basically without pinning, backwards incompatible changes can be inadvertently be foisted upon caravan users in ways that (for example) break signing. This is actually pretty critical as this exact situation has happened before and we had to rush out a patch that did pin versions to avoid users being locked out of funds, so I would hate to lose this.

> I would have assumed the package and connectSrc versions should generally remain in lockstep.

This seems reasonable. Maybe pin the package version in the package.json (remove the `^` and run `npm install` again to update the package-lock) and have the versions match.